src/JsPsych/timeline.js

```diff
@@ -0,0 +1,43 @@
+import { Preamble } from './trials/examples';
+
+/**
+ * Create your custom JsPsych options here. These settings will applied experiment wide.
+ * These options are merged with Honeycomb's own defaults.
+ * https://www.jspsych.org/7.3/overview/experiment-options/
+ */
+export const JSPSYCH_OPTIONS = {
+  // Log each trial on the console
+  on_trial_finish: function (data) {
+    console.log('A trial just ended, here are the latest data:');
+    console.log(data);
+  },
+  //   Sets the default inter-trial interval
+  default_iti: 250,
+};
+
+/**
+ * Build your JsPsych timeline here. The array must be passed as a prop to <JsPsychOptions />
+ * @returns array of trials
+ */
+export function buildTimeline() {
+  const timeline = [
+    Preamble(),
+    // ageCheck,
+    // sliderCheck,
+    // countdown({ message: lang.countdown.message1 }),
+    // taskBlock(practiceBlock),
+    // countdown({ message: lang.countdown.message2 }),
+    // taskBlock(exptBlock1),
+    // demographics,
+    // iusSurvey,
+    // debrief,
+    // showMessage(config, {
+    //   duration: 5000,
+    //   message: lang.task.end,
+    // }),

```

Comment:
is this part of a todo? or can there be an explanation of this to the user?

Reply:
Oh yeah it's worth noting to anyone looking at the PRs that these are all going into a `v4` holding branch, not main. So all the commented out code is things to do that I'm still working on